guides/doc-Upgrading_and_Updating/topics/upgrading_a_connected_satellite_server.adoc

```diff
@@ -13,7 +13,13 @@ If you customize configuration files, manually or using a tool such as Hiera, th
 * On a virtual machine, take a snapshot.
 * On a physical machine, create a backup.
 +
-For more information about backups, see https://access.redhat.com/documentation/en-us/red_hat_satellite/{ProductVersionPrevious}/html/administering_red_hat_satellite/backing-up-satellite-server-and-capsule-server[Backing Up {ProjectServer} and {SmartProxyServer}] in the _Administering {ProjectName} {ProductVersionPrevious}_ guide.
+ifeval::["{ProjectVersionPrevious}" == "2.3"]
+For more information about backups, see {SiteURL}/{ProjectVersion}/administering_red_hat_satellite/index-foreman-el.html#backing-up-satellite-server-and-capsule-server[Backing Up {ProjectServer} and {SmartProxyServer}] in the _Administering {ProjectName} {ProjectVersionPrevious}_ guide.
```

Comment:
While the whole document is very focused on EL, we may want to create an issue to write similar instructions for Debian.